cdap-security/src/main/java/io/cdap/cdap/security/auth/TinkCipher.java

```diff
@@ -59,10 +63,8 @@ public TinkCipher(SConfiguration sConf) throws CipherException {
       this.keysetHandle = CleartextKeysetHandle.read(JsonKeysetReader.withString(jsonKeyset));
 
       this.aead = keysetHandle.getPrimitive(Aead.class);
-    } catch (GeneralSecurityException e) {
-      throw new CipherException("Failed to init Tink cipher: " + e.getMessage(), e);
-    } catch (IOException e) {
-      throw new CipherException("Failed to load Tink keyset: " + e.getMessage(), e);
+    } catch (Exception e) {
+      throw new RuntimeException("Failed to create TinkCipher: " + e.getMessage(), e);
```

Comment:
nit: It's better to use something more specific like `IllegalStateException`

Reply:
Done